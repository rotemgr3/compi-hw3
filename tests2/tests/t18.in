//piazza 87 - should be errorDef (line 2) +
int main() {int booga = 5;}
void main (int a) {int booga = 5;}
void main() {int booga = 5;}
