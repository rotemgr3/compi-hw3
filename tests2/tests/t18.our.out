---end scope---
booga INT 0
line 3: identifier main is already defined
